frontend/preview/spec.html

```diff
@@ -0,0 +1,1549 @@
+        <code>true</code> and the string variable <code>errmsg</code> to a description of the error.
+        The Evy program can then check the value of <code>err</code> and handle the error
+        accordingly. You can trigger a recoverable error in your own code by setting
+        <code>err</code> and <code>errmsg</code>.
+      </p>
+      <p>
+        For more information on run-time panics and recoverable errors, see the built-in
+        documentation on the <a href="builtins.html#panic">panic function</a> and the
+        <a href="builtins.html#errors">errors section</a>.
+      </p>
+      <h2>Execution Model and Event Handlers</h2>
+      <p>
+        Evy first executes all top-level code in the order it appears in the source code. If there
+        is at least one event handler, Evy then enters an event loop. In the event loop, Evy waits
+        for external events, such as a key press or a pointer down event. When an event occurs, Evy
+        calls the corresponding event handler function if it has been implemented. The event handler
+        function can optionally receive arguments, such as the key character or the pointer
+        coordinates. Once the event handler function has finished, Evy returns to the event loop and
+        waits for the next event.
+      </p>
+      <p>
+        Event handlers are declared using the <code>on</code> keyword. Only predefined events can be
+        handled: <code>key</code>, <code>down</code>, <code>up</code>, <code>move</code>,
+        <code>animate</code>, and <code>input</code>. The parameters to the event handlers must
+        match the expected signature. The parameters can be fully omitted or fully specified. If
+        only some parameters are needed, use the anonymous <code>_</code> parameter.
+      </p>
+      <p>
+        For more information on individual event handlers, see the
+        <a href="builtins.html#event-handlers">built-in documentation</a>.
+      </p>
+      <h2>Runtimes</h2>
+      <p>
+        Evy has two runtimes: the <strong>terminal runtime</strong> and the
+        <strong>browser runtime</strong>.
+      </p>
+      <p>
+        The browser runtime can be tried at [play.evy.dev]. It fully supports all built-in functions
+        and event handlers as described in the <a href="builtin.html">built-in documentation</a>.
+      </p>
+      <p>To use the terminal runtime, first install Evy and then run</p>
+      <pre><code>evy run FILE.evy
+</code></pre>
+      <p>
+        in the terminal. This will execute the source code in the given file. You can also use the
+        evy command to format your source code with
+      </p>
+      <pre><code>evy fmt FILE.evy
+</code></pre>
+      <p>
+        For more details, run <code>evy run --help</code> or <code>evy fmt --help</code>. The
+        terminal runtime does not support event handlers or graphics functions.
+      </p>
+      <p><a href="/play">play.evy.dev</a></p>
```

Comment:
looks like this is the link part of the reference-style link that should not be here.